department-of-veterans-affairs/abd-vro
# Add a deployable sleep pod

## What was the problem?

When we removed our postgres deployment, we also lost a pod with psql-client tools installed that we could use to interact with RDS

Associated tickets or Slack threads:
- #2444 

## How does this fix it?[^1]

Adds a new dev-tools container and chart to be deployed to LHDI

## How to test this PR
- Confirmed that the image passes SecRel 
- And confirmed that I can use psql in the pod shell in LHDI


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
